localization/hi/abstract-document/README.md

```diff
@@ -31,7 +31,7 @@ tag:
 प्रकार-सुरक्षा का समर्थन। पैटर्न एक वर्ग के विभिन्न गुणों को अलग-अलग करने के लिए लक्षणों का उपयोग करता है
 इंटरफेस।
 
-**Programmatic Example**
+**प्रोग्रामेटिक उदाहरण**
```

Comment:
Changing this seems appropriate

Reply:
So could you pls merge it to the main branch.